### PR TITLE
feat: customizable tolerations

### PR DESCRIPTION
**Pull request by raelga**
- Enables tolerations customization for the managed exporters

/kind feature
/priority important-soon
/assign

### CODE REVIEW

File: docs/prometheus-exporter-crd-reference.md [hunk 1]
```diff
@@ -48,132 +48,140 @@ spec:
...
+|       `sendgridLocation`        | `string` |      No      |                                           |                      Time zone (empty means UTC)                       |
+|       `sendgridTimeOffet`       |  `int`   |      No      |                    `0`                    | Offset in seconds from UTC as an integer (needed along with location)  |
 
-* Image, port, resources, liveness, readiness default values can be found at [ansible-sendgrid-vars](../roles/prometheusexporter/exporters/sendgrid/vars.yml)
-* Real `sendgrid` example can be found on [examples](../examples/README.md#sendgrid) directory.
\ No newline at end of file
+- Image, port, resources, liveness, readiness default values can be found at [ansible-sendgrid-vars](../roles/prometheusexporter/exporters/sendgrid/vars.yml)
+- Real `sendgrid` example can be found on [examples](../examples/README.md#sendgrid) directory.
+
+```
```
Comment (slopezz) [inline docs/prometheus-exporter-crd-reference.md:185]: why this new block?

Comment (raelga) [reply docs/prometheus-exporter-crd-reference.md]: No idea, removed in https://github.com/3scale-ops/prometheus-exporter-operator/pull/59/commits/f2b7b5d1ebc8c05c298ed56523f0a426d3824c21

**Pull request merged**